adminservice/src/main/java/nu/yona/server/AdminServiceApplication.java

```diff
@@ -38,8 +40,11 @@
 @SpringBootApplication
 @EnableSwagger2
 @EnableHypermediaSupport(type = HypermediaType.HAL)
+@EnableCaching
 public class AdminServiceApplication
 {
+	@Autowired
+	CacheManager cacheManager;
```

Comment:
Looking even further :)
Why do we need to create a cache manager ourselves? Spring Boot does that automatically, right? See [this article](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-caching.html) and also [this one](https://spring.io/blog/2014/06/16/further-cache-improvements-in-spring-4-1).